src/emailagent.h

```diff
@@ -44,9 +44,9 @@ class Q_DECL_EXPORT EmailAgent : public QObject
         Unknown,
         Queued,
         Downloading,
-        // following are transient states within emailagent. I.e. download finished will be signaled, but not remembered
         NotDownloaded,
         Downloaded,
+        // following are transient states within emailagent. I.e. download finished will be signaled, but not remembered
```

Comment:
If moving the line should now indicate the above are no longer transient, the text should probably be adjusted not to mention download finished.

Reply:
Good point !